packages/backend/src/tests/generated/db.ts

```diff
@@ -57,6 +57,13 @@ export type PgCatalogNumeric = string;
  * The mpaa_rating enum from the public schema
  */
 export type PublicMpaaRatingEnum = 'G' | 'PG' | 'PG-13' | 'R' | 'NC-17';
+/**
+ * A PG text
+ */
+export type PgCatalogText1 = {
```

Comment:
this is really suspicious 🤔

Reply:
Indeed. I've reworked this now, but we're still WIP